src/main/java/org/kiwiproject/jersey/client/dropwizard/DropwizardManagedClientBuilder.java

```diff
@@ -147,4 +131,33 @@ public RegistryAwareClient buildManagedRegistryAwareClient() {
         var jerseyClient = buildManagedJerseyClient();
         return new RegistryAwareClient(jerseyClient, registryClient);
     }
+
+    /**
+     * Creates a new {@link JerseyClientConfiguration} to use as a default that is <em>NOT</em> configured for TLS.
+     *
+     * @return a {@link JerseyClientConfiguration} with defaults set
+     */
+    public static JerseyClientConfiguration newDefaultJerseyClientConfiguration() {
+        return newDefaultJerseyClientConfiguration(null);
+    }
+
+    /**
+     * Creates a new {@link JerseyClientConfiguration} to use as a default that has TLS configured <em>IF</em> the given
+     * provider is not null and can provide.
+     *
+     * @param tlsConfigProvider a {@link TlsConfigProvider} used to provide TLS settings
+     * @return a {@link JerseyClientConfiguration} with defaults set
+     */
+    public static JerseyClientConfiguration newDefaultJerseyClientConfiguration(TlsConfigProvider tlsConfigProvider) {
```

Comment:
```suggestion
    public static JerseyClientConfiguration newDefaultJerseyClientConfiguration(@Nullable TlsConfigProvider tlsConfigProvider) {
```

_Note you will need to add the import for `@Nullable` which the suggestion won't do!_